lib/mixlib/install/product.rb

```diff
@@ -269,6 +286,8 @@ def lookup(key, version = :latest)
       v < version_for("0.7.0") ? "delivery-ctl" : "automate-ctl"
     end
     config_file "/etc/delivery/delivery.rb"
+    github_repo "chef/automate"
+    downloads_url "https://downloads.chef.io/automate"
```

Comment:
This is still a standalone product which is listed separate on downloads.chef.io:
https://downloads.chef.io/compliance